source/grid/grid_in.cc

```diff
@@ -34,6 +34,30 @@
 #include <netcdfcpp.h>
 #endif
 
+#ifdef DEAL_II_WITH_ASSIMP
+DEAL_II_DISABLE_EXTRA_DIAGNOSTICS
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Waddress-of-packed-member"
+#include <assimp/Importer.hpp>      // C++ importer interface
+#include <assimp/scene.h>           // Output data structure
+#include <assimp/postprocess.h>     // Post processing flags
+#pragma GCC diagnostic pop
+DEAL_II_ENABLE_EXTRA_DIAGNOSTICS
+
+#undef AI_CONFIG_PP_RVC_FLAGS
+#define AI_CONFIG_PP_RVC_FLAGS           \
```

Comment:
could this be done with a local variable in the function that uses it, instead of a global preprocessor define?